frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -611,6 +611,26 @@ const PersonForm = (props: Props) => {
             </div>
           ) : null}
         </div>
+        <div className="usa-form" style={{ maxWidth: "30rem" }}>
+          <div className="usa-form-group">
+            <Label htmlFor="patient-notes-textarea">
+              {t("patient.form.notes.heading")}
+            </Label>
+            <span id="with-hint-textarea-hint" className="usa-hint">
+              {t("patient.form.notes.helpText")}
+            </span>
+            <Textarea
+              id="patient-notes-textarea"
+              maxLength={10000}
+              aria-describedby="with-hint-textarea-info with-hint-textarea-hint"
+              onChange={(value) => {
+                onPersonChange("notes")(value.target.value);
+              }}
+              name="notes"
+              value={patient.notes || undefined}
```

Comment:
Should we default to empty string instead of undefined?

Reply:
it will then populate the db with an empty string vs keeping it as null when no value is provided